### DISCUSSION TITLE
Why is `character(len=*)` underlined as an error

### DISCUSSION THREAD

**Discussion by blaylockbk** [krvajal/vscode-fortran-support#574] [Q&A - deprecated see Fortran Discourse]
I have a subroutine in a large code base. Why do I get an underlined error indicator for `character(len=10)`. When I hover over it, the message says `Object "len" not found in scope` and then gives a file where the variable len is declared as an object. 
![image](https://user-images.githubusercontent.com/6249613/179306013-995d306b-f651-4ad2-9c84-c4997ea4176d.png)

The "len" statemen inside a character declaration would never refer to a declared variable named len, right? Is this a possible bug in fortls, or is there a Fortran feature I don't understand here? This might be a silly concern, but I'd like to figure out how to resolve this underlined error indicator.

**Comment by gnikit** [comment]
This is actually a bug from `fortls` that I have been encountering from time to time when I had variables named `len`.

If you don't mind can you file a bug report in the fortls repo and I'll try and get to it this weekend.
